flut_news/lib/screens/categories_screen.dart

```diff
@@ -24,46 +26,83 @@ class _CategoriesScreenState extends State<CategoriesScreen> {
     var newsData = await NewsModel().getNewsByCategory(category);
     if (newsData != null && newsData['articles'] is List) {
       Navigator.push(context, MaterialPageRoute(builder: (context) {
-        return NewsListScreen(newsData: newsData, category: category); // Pass the whole response
+        return NewsListScreen(
+            newsData: newsData, category: category); // Pass the whole response
       }));
     } else {
       print('No or invalid news data for category: $category');
     }
   }
 
-
-
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       backgroundColor: kIsDarkMode ? Colors.black : Colors.white,
       appBar: AppBar(
         backgroundColor: kIsDarkMode ? Colors.black : Colors.white,
-        title: Text("News Categories", style: TextStyle(fontFamily: 'Merriweather'),),
+        title: Text(
+          "FlutNews",
+          style: TextStyle(
+              fontFamily: 'PlayfairDisplay',
+              fontWeight: FontWeight.w800,
+              letterSpacing: 1.5,
+              color: kIsDarkMode == false
+                  ? const Color.fromARGB(255, 233, 218, 90)
+                  : Colors.teal),
+        ),
         centerTitle: true,
+        automaticallyImplyLeading: false,
+        actions: [
+          IconButton(
+            onPressed: () {
+              setState(() {
+                kIsDarkMode = !kIsDarkMode;
+              });
+            },
+            icon: Icon(
+              kIsDarkMode == false ? Icons.dark_mode : Icons.light_mode,
+              color: kIsDarkMode == false ? Colors.grey : Colors.yellow,
+            ),
+          ),
+        ],
       ),
-      body: ListView.builder(
```

Comment:
builder ı neden sildiğimizi anlamadım

Reply:
silinmemiş eklediğim şeyler vardı ve container içine aldım resmi eklemek için aşağıda

Reply:
eğer fark edemezsen işaretlerim